src/app/src/components/ClaimedFacilitiesDetails.jsx

```diff
@@ -473,9 +473,8 @@ function ClaimedFacilitiesDetails({
                     disabled={updating}
                     isSelect
                     isMultiSelect
-                    isCreatable
                     selectOptions={sectorOptions || []}
-                    selectPlaceholder="e.g. Apparel - Use <Enter> or <Tab> to add multiple values"
+                    selectPlaceholder="Select..."
```

Comment:
As I understand well the task scope is only update placeholder text? @Innavin369